src/test/test-utils.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {Server} from 'http';
+import Koa from 'koa';
+import route from 'koa-route';
+import {getPort} from 'portfinder';
+import {Logger} from '../support/logger';
+
+export type AppOptions = {
+  middleware?: Koa.Middleware[],
+  routes?: {[key: string]: string|Function},
+};
+
+export const createApp = (options: AppOptions): Koa => {
+  const app = new Koa();
+  const {middleware, routes} = options;
+  if (middleware) {
+    for (const m of middleware) {
+      app.use(m);
+    }
+  }
+  if (routes) {
+    for (const key of Object.keys(routes)) {
+      const value = routes[key];
+      app.use(route.get(key, (ctx) => {
+        if (key.endsWith('.js')) {
+          ctx.type = 'js';
+        }
+        if (key.endsWith('.html')) {
+          ctx.type = 'html';
+        }
+        ctx.body = value;
```

Comment:
I recommend doing the same thing from this commit https://github.com/Polymer/koa-node-resolve/commit/a02e72107acfc08fec40f76d80422268aa0c612c#diff-74e5fabe1476390873e3015312f17489 and making the body a stream, like it would be with a real koa static server middleware.

Reply:
Ah okay.  Cool :) Done.